openwisp-monitoring/tests/test_ula.lua

```diff
@@ -0,0 +1,33 @@
+package.path = package.path .. ";../files/lib/openwisp-monitoring/?.lua" ..
```

Comment:
please rename it to `test_ula_prefix.lua` and make sure it's added to the `runtests` script otherwise it will not be run during tests.